### PR TITLE
Added feed status component

### PR DESCRIPTION
**Pull request by robineng**

### CODE REVIEW

**Comment by robineng** [comment]
Related to #198

File: src/components/freshness-indicator/freshness-indicator-tooltip-delay-content.jsx [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  time.seconds = currDelay;
+  return time;
+}
+
+function getTimeComponent(time, key) {
+  if (!time[key]) {
+    return null;
+  }
+  return (
+    <span key={key} >{time[key]} <FormattedMessage {...messages[key]} /> </span>
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator-tooltip-delay-content.jsx]: My hunch is that **3h 5m 23s** is better than **3 h 5 m 23 s**

File: src/components/freshness-indicator/freshness-indicator-tooltip-delay-content.jsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    id: 'COMPONENT_KIT.FRESHNESS_INDICATOR.MINUTE',
+    defaultMessage: 'm',
+    description: 'Suffix for delay minute value in feed tooltip',
+  },
+  seconds: {
+    id: 'COMPONENT_KIT.FRESHNESS_INDICATOR.SECOND',
+    defaultMessage: 's',
+    description: 'Suffix for delay second value in feed tooltip',
+  },
+  isActive: {
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator-tooltip-delay-content.jsx]: `isActive` -> `notActive`

File: src/components/freshness-indicator/freshness-indicator-tooltip-delay-content.jsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  delay: 0,
+  isActive: false,
+};
+
+FreshnessIndicatorTooltipDelayContent.propTypes = {
+  delay: PropTypes.number,
+  isActive: PropTypes.bool,
+};
+
+export default FreshnessIndicatorTooltipDelayContent;
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator-tooltip-delay-content.jsx]: Maybe change `FreshnessIndicatorTooltipDelayContent` to `TooltipDelayContent`?

File: src/components/freshness-indicator/freshness-indicator-tooltip-delay.jsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import React, { PropTypes } from 'react';
+import FreshnessIndicatorTooltipDelayContent from './freshness-indicator-tooltip-delay-content';
+
+
+function FreshnessIndicatorTooltipDelay(props) {
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator-tooltip-delay.jsx]: Maybe remove FreshnessIndicator from this component name?

File: src/components/freshness-indicator/freshness-indicator-tooltip-timestamp.jsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+const messages = defineMessages({
+  updated: {
+    id: 'COMPONENT_KIT.FRESHNESS_INDICATOR.UPDATED',
+    defaultMessage: 'Updated',
+    description: 'Label for timestamp value in feed status tooltip',
+  },
+});
+
+function FreshnessIndicatorTooltipTimestamp({ timestamp }) {
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator-tooltip-timestamp.jsx]: See comment on FreshnessIndicatorTooltipDelay

File: src/components/freshness-indicator/freshness-indicator-tooltip-timestamp.jsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+  if (!Number.isFinite(timestamp)) {
+    return (<div />);
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator-tooltip-timestamp.jsx]: `return null`

File: src/components/freshness-indicator/freshness-indicator-tooltip-timestamp.jsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+  }
+  return (
+    <div className="freshness-indicator__tooltip__timestamp">
+      <b><FormattedMessage {...messages.updated} />: <DateTime value={new Date(timestamp)} format="numeric" type="time" /></b>
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator-tooltip-timestamp.jsx]: You should be able to omit `new Date()` from the value and instead pass `value={timestamp}`.

File: src/components/freshness-indicator/freshness-indicator.examples.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+## Translations
+
+ID | Default
+------------ | -------------
+COMPONENT_KIT.FRESHNESS_INDICATOR.DELAY | Delayed
+COMPONENT_KIT.FRESHNESS_INDICATOR.NO_DELAY | Realtime
+COMPONENT_KIT.FRESHNESS_INDICATOR.HOUR  | h
+COMPONENT_KIT.FRESHNESS_INDICATOR.MINUTE | m
+COMPONENT_KIT.FRESHNESS_INDICATOR.SECOND | s
+COMPONENT_KIT.FRESHNESS_INDICATOR.IS_ACTIVE | Close price
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator.examples.md]: Don't forget to update key

File: src/components/freshness-indicator/freshness-indicator.examples.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+COMPONENT_KIT.FRESHNESS_INDICATOR.UPDATED | Updated
+
+No delay feed:
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator.examples.md]: delay*ed data*?

File: src/components/freshness-indicator/freshness-indicator.examples.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+
+    <FreshnessIndicator delay={0} timestamp={1} />
+
+Delay feed:
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator.examples.md]: `**Delayed data:**`

File: src/components/freshness-indicator/freshness-indicator.jsx [hunk 11]
```diff
@@ -0,0 +1,87 @@
+import React, { PropTypes } from 'react';
+import { Icon, variables } from 'nordnet-ui-kit';
+import ReactTooltip from 'react-tooltip';
+import uuid from 'uuid';
+import classNames from 'classnames';
+import FreshnessIndicatorTooltipTimestamp from './freshness-indicator-tooltip-timestamp';
+import FreshnessIndicatorTooltipDelay from './freshness-indicator-tooltip-delay';
+
+
+function getIconType(delay, isActive) {
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator.jsx]: export and test separately

File: src/components/freshness-indicator/freshness-indicator.jsx [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+  if (isActive) {
+    return 'circleSlash';
+  }
+  if (delay === 0) {
+    return 'lightningBolt';
+  }
+  return 'tickingClock';
+}
+
+function getColor(isActive) {
```
Comment (bstream) [inline src/components/freshness-indicator/freshness-indicator.jsx]: export and test separately

File: test/components/freshness-indicator/freshness-indicator-tooptip-delay-content.test.js [hunk 13]
```diff
@@ -0,0 +1,32 @@
...
+
+  it('should render 3 formatted messages if the delay is 61 sec', () => {
+    expect(getComponent(61).find('FormattedMessage')).to.have.length(3);
+  });
+
+  it('should render 4 formatted messages if the delay is 60 * 61 + 1 sec', () => {
+    expect(getComponent((60 * 61) + 1).find('FormattedMessage')).to.have.length(4);
+  });
+
+  it('should render DELAY message if isActive is larger than 1 and isActive is false', () => {
```
Comment (bstream) [inline test/components/freshness-indicator/freshness-indicator-tooptip-delay-content.test.js]: `should render DELAY message if isActive is larger than 1 and isActive is false` -> `should render DELAY message if delay is larger than 1 and isActive is false`

**Review by bstream — CHANGES_REQUESTED** [review]
Please see the comments below 🚀

**Review by bstream — APPROVED** [review]

**Comment by bstream** [comment]
Well done 🙏

**Pull request merged**